dotcom-rendering/src/lib/decideContainerOverrides.ts

```diff
@@ -43,44 +46,49 @@ const textCardHeadline = (
 const textCardStandfirst = textCardHeadline;
 const textCardFooter = textCardHeadline;
 
-const textContainerFooter = (
-	containerPalette: Extract<DCRContainerPalette, 'Branded'>,
-): string => {
+const textContainerFooter = (containerPalette: DCRContainerPalette): string => {
 	switch (containerPalette) {
-		case 'Branded':
+		case 'BrandedPalette':
+			return neutral[46];
+		default:
 			return neutral[46];
 	}
 };
 
 const textContainerSummary = (
-	containerPalette: Extract<DCRContainerPalette, 'Branded'>,
+	containerPalette: DCRContainerPalette,
 ): string => {
 	switch (containerPalette) {
-		case 'Branded':
+		case 'BrandedPalette':
 			return labs[400];
+		default:
+			return neutral[46];
 	}
 };
 
 const backgroundContainerLeftColumn = (
-	containerPalette: Extract<DCRContainerPalette, 'Branded'>,
+	containerPalette: Extract<DCRContainerPalette, 'BrandedPalette'>,
```

Comment:
I'm not 100% sure why these `Extract` type helpers are used, and they seem to be used differently in different situations. Maybe worth looking at in another PR to not distract from the point of this PR.

Reply:
Discussed with @alinaboghiu and a potential refactor could use `as const` assertions in a follow-up PR:

```ts
const textCardHeadlineObj = {
	LongRunningPalette: neutral[100],
	LongRunningAltPalette: neutral[7],
	SombrePalette: neutral[100],
	SombreAltPalette: neutral[100],
	InvestigationPalette: neutral[100],
	BreakingPalette: neutral[100],
	EventPalette: brand[300],
	EventAltPalette: brand[300],
	SpecialReportAltPalette: specialReportAlt[100],
	BrandedPalette: undefined,
	MediaPalette: undefined,
} as const satisfies Record<DCRContainerPalette, string | undefined>;
```

Reply:
we have agreed to do this as a follow up refactor - I think it's a lot neater so thank you for this suggestion